src/api/transactionSubmission/simulate.ts

```diff
@@ -51,10 +51,10 @@ export class Simulate {
    */
   @ValidateFeePayerDataOnSimulation
   async multiAgent(args: {
-    signerPublicKey: PublicKey;
+    signerPublicKey: PublicKey | null;
     transaction: AnyRawTransaction;
-    secondarySignersPublicKeys: Array<PublicKey>;
-    feePayerPublicKey?: PublicKey;
+    secondarySignersPublicKeys: Array<PublicKey | null>;
```

Comment:
I proposed `secondarySignersPublicKeys: Array<PublicKey | null>` because some secondary signers may provide public keys for auth key check, while others may not. For example, `secondarySignersPublicKeys` can be `[publicKey1, null, publicKey3, null]` where only first and third secondary signers provided public keys.

Reply:
Updated it to `secondarySignersPublicKeys: Array<PublicKey | undefined>;`